beetsplug/lyrics.py

```diff
@@ -693,6 +738,13 @@ def __init__(self):
             )
             sources.remove('genius')
 
+        if 'tekstowo' in sources and not HAS_BEAUTIFUL_SOUP:
+            self._log.debug(
+                u'The Tekstowo.pl backend requires BeautifulSoup, which is not '
+                u'installed, so the source is disabled.'
+            )
+            sources.remove('tekstowo')
```

Comment:
Since we now have three different sources that all need the same check and message, maybe it's time to factor this out into a function so we don't need to copy n' paste the error message?